command/task_create.go

```diff
@@ -265,7 +264,8 @@ func handleDeprecations(ui mcli.Ui, tc *config.TaskConfig) error {
 		if tc.Condition != nil {
 			switch tc.Condition.(type) {
 			case *config.ServicesConditionConfig:
-				action := generateServicesConditionBlockAction(tc.Services)
+				action := `"list of 'services' and 'condition "services"' block cannot both be configured. ` +
```

Comment:
It seems like the double-quotes at the start of the string might not be necessary